### PR TITLE
Fix deprecated syntax, bump to Ansible 2.2

### PR DESCRIPTION
**Pull request by manics**
Min is 2.2, ~~but I thought I'd bump to 2.3 since that's what we'll be testing~~

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ galaxy_info:
   description: Upgrades all packages installed with the distribution's package manager
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.3
```
Comment (sbesson) [inline meta/main.yml]: My understanding from last week is that we agreed on having Ansible 2.2 as the minimum version for our public roles but that we were going to use Ansible 2.3 internally in particular for the credentials managements functionalities.

Would bumping the minimal version in this role effectively mean bumping the minimal Ansible version across the board?

Comment (manics) [reply meta/main.yml]: I don't know....does anything use `min_ansible_version` in the role metadata or is it purely informative?

**Comment by manics** [comment]
@sbesson I've force-pushed the previous commit away so the min is now 2.2. I've also added a molecule test.

**Comment by joshmoore** [comment]
:+1: merging and tagging as 1.0.1

**Pull request merged**